core/common/src/test/java/alluxio/util/SecurityUtilsTest.javaSecurityUtilsTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.util;
+
+import alluxio.ConfigurationTestUtils;
+import alluxio.conf.InstancedConfiguration;
+import alluxio.conf.PropertyKey;
+import alluxio.security.LoginUserTestUtils;
+import alluxio.security.authentication.AuthType;
+import alluxio.security.authentication.AuthenticatedClientUser;
+import alluxio.security.group.provider.IdentityUserGroupsMapping;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import static  org.junit.Assert.assertEquals;
```

Comment:
leave line 25 an empty line and move `import static  org.junit.Assert.assertEquals;` before all other import blocks